src/chart_types/xy_chart/utils/series.ts

```diff
@@ -182,14 +182,12 @@ export function splitSeriesDataByAccessors(
     let sum = xValueSums.get(x) ?? 0;
 
     // extract small multiples aggregation values
-    const smH = smallMultiples?.horizontal?.by
-      ? smallMultiples.horizontal?.by(spec, datum)
-      : DEFAULT_SINGLE_PANEL_SM_VALUE;
+    const smH = smallMultiples?.horizontal?.by?.(spec, datum) ?? null;
```

Comment:
nit: we can keep using undefined also here and 5 lines below
```suggestion
    const smH = smallMultiples?.horizontal?.by?.(spec, datum);
```

Reply:
3c175e1

Reply:
awesome, it just probably needs fixing the snapshot tests

Reply:
good to merge on green